test/compression-test.js

```diff
@@ -3,40 +3,47 @@
  * MIT License <https://github.com/level/leveldown/blob/master/LICENSE.md>
  */
 
-var async = require('async'),
-  du = require('du'),
-  delayed = require('delayed'),
-  common = require('abstract-leveldown/testCommon'),
-  leveldown = require('..'),
-  test = require('tape'),
+const async = require('async')
+const du = require('du')
+const delayed = require('delayed')
+const common = require('abstract-leveldown/testCommon')
+const leveldown = require('..')
+const test = require('tape')
 
-  compressableData = Buffer.from(Array.apply(null, Array(1024 * 100)).map(function () { return 'aaaaaaaaaa' }).join('')),
-  multiples = 10,
-  dataSize = compressableData.length * multiples,
+const compressableData = Buffer.from(Array.apply(null, Array(1024 * 100)).map(function () {
```

Comment:
Side note: `Array.apply(null, Array(1024 * 100)).map(..)` seems overly complicated. Maybe we can use `array.fill`  instead?